common/changes/@uifabric/experiments/fonts_theme_2018-10-10-23-23.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "updating snapshot tests in experiments package",
+      "type": "patch"
```

Comment:
If the changes are truly snapshot tests only (e.g. you added new tests/you changed an existing test), it should be none. If the snapshots changed without any test code changes though, it probably means there was a real change and the comment should capture the real change that caused the snapshots to change

Reply:
Ah I misunderstood where the snapshots were being updated - I'll change the comment when I run npm change again after I fix the builds/screener changes!